src/Simplex/Messaging/Transport.hs

```diff
@@ -134,11 +141,14 @@ data ATransport = forall c. Transport c => ATransport (TProxy c)
 
 -- * Transport over TLS 1.2
 
+-- | Party name to print on exception handling - for debugging.
+newtype PartyAlias = PartyAlias {partyAlias :: String} deriving (Show)
+
 -- | Run transport server (plain TCP or WebSockets) on passed TCP port and signal when server started and stopped via passed TMVar.
 --
 -- All accepted connections are passed to the passed function.
-runTransportServer :: forall c m. (Transport c, MonadUnliftIO m) => TMVar Bool -> ServiceName -> T.ServerParams -> (c -> m ()) -> m ()
-runTransportServer started port serverParams server = do
+runTransportServer :: forall c m. (Transport c, MonadUnliftIO m) => PartyAlias -> TMVar Bool -> ServiceName -> T.ServerParams -> (c -> m ()) -> m ()
```

Comment:
@epoberezkin could be just String [instead of PartyAlias] but I thought it's better to give it a name since it's part of API, do you see drawbacks?

Also it can be removed altogether but then it's hard to figure out what exactly fails on tests

Reply:
If you want name in the API, it could be just type synonym - this is better:

```
type PartyAlias = String -- or ByteString, still don't see where it is used
```